### DISCUSSION TITLE
Phenotypes comparison in C. intermedia (Leloir vs. oxido-reductive pathway)

### DISCUSSION THREAD

**Discussion by IVANDOMENZAIN** [IVANDOMENZAIN/diverseYeasts_metabolism#6] [Ideas]
Hi @abrilvalras and @kdeperi, I just found out this new functionality of GitHub, it seems that `GitHub-discussions` is a more friendly way of sharing ideas (not necessarily code-related) about our project. Let's start this with the discussion about phenotypic differences for *C. intermedia* when growing on lactose by using either the Leloir or the Oxido-reductive pathway. This discussion is linked to the issue #3 (opened in our last meeting), but let's try to write our ideas here, no matter how developed they are :)

**Comment by IVANDOMENZAIN** [comment]
So far @abrilvalras has proposed a FSEOF-like approach to explore how both pathways contribute to growth, If I understood you correctly, does this mean to try a series of optimizations for biomass production subject to incremental levels of lactose uptake rate at each step, then with this we could identify the reactions with fluxes that are consistently being increased at each of the steps?

**Reply by abrilvalras** [reply]
Exactly! Maybe if we use a very Carbon-limited medium and open the lactose gates little by little we might have an idea of how the cell chooses to distribute the fluxes with any or both pathways and we might get an idea of why the OR pathway isn't as efficient as the Leloir pathway in making biomass.

**Reply by IVANDOMENZAIN** [reply]
Nice! I agree, very thorough and even quantitative way to assess this. Let's start the work on this. I've just created a [branch (link here)](https://github.com/IVANDOMENZAIN/diverseYeasts_metabolism/tree/feat/lactose_pathways_comparison) for this specific task, feel free to add any code that you've already tried there @abrilvalras or just tell us if you prefer to start from scratch :)

PS. Also, I worked a bit on the structure of the repo, now we have a proper README, contribution guidelines and an organized branching system.

**Reply by kdeperi** [reply]
Very nice idea indeed! Could surely get very good insights about both the pathways.

Also, this complements one of the experimental setup which me and Cecilia want to try for a bachelor students' thesis project in spring. We would like to optimise synthetic media (representing cheese whey) for growth rate and production of growth related metabolites, using a design expert software-MODE. So during this check, we will indeed end up checking different concentrations of lactose in an incremental order. If everything goes well then, we can get samples for qPCR analysis, thus connecting to gene expression patterns in both pathways at different concentrations of lactose.  I will see if minimal media and mutants can be taken along or not, thus experimentally validating the results in the model.

Further, about the question regarding lower growth rate: So as far as I remember, we deleted galactokinase  gene and the growth rate dropped slightly. This indicates that leloir pathway has a certain contribution to the overall growth rate = makes sense! Next step could be to delete Xyl1 (xyl1 mutant) and then see how the growth rate is effected (if we already did this, please correct me - experimentally, the growth is effected quite significantly for this mutant), then the contribution of oxido-reductive pathway would be visible. @abrilvalras  I didnt completely understand your point about the L-xylo-3-hexulose reductase regulation that you had mentioned in the mail. 

Also, one more thing to check...to differentiate alpha-d-galactose from b-d-galactose in the model because of the specificity of xyl1 to alpha and beta-d-galactose and galactokinase for alpha-d-galactose. Ok. Too much here. Will try sharing the pathway scheme. 

![Picture 1](https://user-images.githubusercontent.com/67782953/101946845-cf88be80-3bef-11eb-8418-5434a6b94b85.png)

**Reply by IVANDOMENZAIN** [reply]
Great to have a diagram here for future references @kdeperi!

**Comment by kdeperi** [comment]
Powerpoint presentation showing the overall structure of the project. 
[CintermediaGEM_KPeri_04122020.pptx](https://github.com/IVANDOMENZAIN/diverseYeasts_metabolism/files/5716178/CintermediaGEM_KPeri_04122020.pptx)

**Comment by IVANDOMENZAIN** [comment]
Updated FSEOF results (leloir vs. oxido-reductive) [here](https://github.com/IVANDOMENZAIN/diverseYeasts_metabolism/blob/feat/FSEOF/results/lactose_pathways_comparison_Cint.txt) @kdeperi
